### ISSUE TITLE
KeyError: 'ImageId' on running full_scan.py

### ISSUE THREAD

**Issue by creatorrr** [codemonauts/aws-springcleaning#16]
`config.py` contents:

```python
#! /usr/bin/env python3

# Regions to crawl for resources
REGIONS = ["us-east-1", "us-west-1", "ap-south-1"]

# Alert when EC2 instances run for this amount of days
EC2_OLD_DAYS = 100

# Alert when EC2 snapshots are this old
EC2_SNAPSHOT_OLD_DAYS = 100

# Alert when RDS snapshots are this old
RDS_SNAPSHOT_OLD_DAYS = 100

# Alert when Cloudformation Stacks haven't been updated in this amount of days
CF_STACK_OLD_DAYS = 100
```

**Comment by fleaz** [comment]
Hey @creatorrr 

can you please provide the full stacktrace of the error?

**Comment by fleaz** [comment]
Fixed in 06f404d2ef558fe674133306dd5b448f8936e691
